gimme

```diff
@@ -176,7 +176,7 @@ _binary() {
 		# attempt "armv6l" vs just "arm" first (since that's what's officially published)
 		urls=(
 			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}v6l.tar.gz" # go1.6beta2 & go1.6rc1
-			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}6.tar.gz" # go1.6beta1
+			"${GIMME_DOWNLOAD_BASE}/go${version}.${GIMME_OS}-${arch}6.tar.gz"   # go1.6beta1
```

Comment:
oh my bad this was shfmt hooked up with my editor. let me know if I should revert this change.